### ISSUE TITLE
Dependabot V2 behaviour on PR merge

### ISSUE THREAD

**Issue by ElvenSpellmaker** [dependabot/dependabot-core#3890]
Previously, using `app.dependabot.com` we had automatic rebasing disabled because otherwise we're swamped with dependabot builds on every PR merge.
However, when we merged a PR, dependabot would close any PRs that had been solved in the PR we just merged, whereas v2 dependabot only seems to look at HEAD (not the rebased HEAD) and so never closes any PRs.

This means we have to go to _each_ PR and `@dependabot rebase` on each one which is really time consuming and slow.

Basically the behaviour before where Dependabot would do its checks vs the rebased HEAD (without actually rebasing HEAD on our PR) was much better behaviour.

**Comment by ElvenSpellmaker** [comment]
Another side-effect of having the rebase policy set to disabled is it seems Dependabot doesn't monitor for changes anymore in versions and close and superceed versions like it used to

**Comment by asciimike** [comment]
I believe this is the same request as https://github.com/dependabot/dependabot-core/issues/3880, yes?

**Comment by ElvenSpellmaker** [comment]
@asciimike Thanks, sorry I couldn't find the issue when I searched, it does seem that's a similar/same issue!

**Comment by asciimike** [comment]
Sounds good, closing as a dupe, and we're hoping to look into rebasing behavior and either document the current behavior better or tweak it (I believe we stopped rebasing everything by default since it was thrashing actions and burning folks' minutes, but if folks want that behavior, we may expose the option for folks to rebase everything if desired).

**Comment by ElvenSpellmaker** [comment]
Hey @asciimike, you're conflating two different issues. Rebasing the _actual_ PR is annoying as it burns minutes or in our cases using too many agents in Azure DevOps, however Dependabot used to rebase on its end and not update the PR.

Basically checks for dependencies are run against a _local rebase_ branch and never pushed to the PR.
